### DISCUSSION TITLE
Dependencies on Nuget seem to be broken in the latest version

### DISCUSSION THREAD

**Discussion by Calandiel** [SixLabors/ImageSharp#1327] [Q&A] [answered]
I just added ImageSharp to my project (version 1.0.1) and it appears to me that the dependencies are somehow wrong?
According to this: [https://www.nuget.org/packages/SixLabors.ImageSharp/1.0.0](https://www.nuget.org/packages/SixLabors.ImageSharp/1.0.0), it depends on System.Buffers (>= 4.5.1). But having inspected the dll, it appears to depend specifically on version 4.0.1.

Did anyone else notice this issue too? I'm not 100% sure if I'm not the one at fault here.

**Comment by JimBobSquarePants** [comment, marked as answer]
Sounds like you’re on .NET Framework and suffering from a binding redirect issue.

https://nickcraver.com/blog/2020/02/11/binding-redirects/
